feat(connect-examples): new webextension example for mv3 sw and ES6 modules in TypeScript

karliatto opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

New webextension example for mv3 sw and ES6 modules in TypeScript.

This will make it more clear to developers using ES6 modules and Typescript and closer to a realistic realworld application scaffolding. 
At the same time it provides a way for us to test webextension package with ES6 modules.

martykan commented:
Looks good, just a minor detail, it should probably be added to `connect-examples` README and maybe also linked in `connect-webextension` README

karliatto commented:
> Looks good, just a minor detail, it should probably be added to `connect-examples` README and maybe also linked in `connect-webextension` README

I added here to connect-example README [e6378a6](https://github.com/trezor/trezor-suite/pull/12987/commits/e6378a67e7c39aeb8b4a98dee97e2fac40c25871)
And in connect-webextension I added it by removing connect-explorer since I think that connect-explorer webextension can be very confusing for developers, but you might have different opinion, let me know if so.  [b0855c6](https://github.com/trezor/trezor-suite/pull/12987/commits/b0855c643da7ea7ee10609cb74e8c5844d20cffd)
